bookserver/config.py

```diff
@@ -81,11 +80,16 @@ def database_url(self) -> str:
     web2py_path: str = str(Path(__file__).parents[2] / "web2py")
 
     # This is the private key web2py uses for hashing passwords.
-    @property  # type: ignore
-    @lru_cache
+    @property
     def web2py_private_key(self) -> str:
-        with open(self.web2py_path, encoding="utf-8") as f:
-            return f.read()
+        authfile = Path(self.web2py_path) / "private" / "auth.key"
+        with open(authfile, encoding="utf-8") as f:
+            return f.read().strip()
+
+    # web2py_private_key: str = "sha512:16492eda-ba33-48d4-8748-98d9bbdf8d33"
+    # if you want to reinitialize your database set this to Yes
+    # All data in the database will be lost
+    drop_tables: str = "No"
```

Comment:
This seems like a testing-only setting -- it seems like a dangerous thing.

Why not make this a `bool`?

Reply:
I wanted to make it hard to be true

Reply:
Maybe the condition should be:

```
if settings.book_sever_config in ["development", "test"] and settings.drop_tables == "Yes":
```
This would rule out the accidental dropping of tables in a production environment.

Reply:
I like it.